src/phrases/phrases-en.ts

```diff
@@ -217,6 +217,7 @@ const phrases = {
     newConcept: 'Create new concept',
     newSubject: 'Create new subject',
     newFrontpageArticle: 'Create new content article',
+    creationMovedInfo: 'Topics and resources need to be created from structural edit',
```

Comment:
Har vi et ordentlig begrep for struktur på engelsk?

Reply:
Det er vel eegentlig bare 'structure' som brukes, så oppdaterer til det!